.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
I'm pretty annoyed of pre-commit hooks; do you find them really useful? 

In my workflow I generally run tests in watch mode and so when I commit I already know that the tests are going to pass; also pre-commit hooks are extremely annoying when having to tag the service.

I'm more on the opinion to dismiss them, what is yours?

Reply:
I think it is useful to have a default check before committing and tagging. While using watching mode is viable, new template adopters might not expect to run it.

One possible option could be to leave pre-commit configuration, but do not execute hook installation by default. In this manner, who would use the pre-commit script can do so (maybe using a custom `npm` script instead of `prepare`) while who prefer to not run pre-commit can skip that stage.

Reply:
Ok, this solution seems good to me

Reply:
IMO pre-commit hooks should only lint, and not tests.

Tests should be made by pipelines.